src/Assets/Bugsnag/Editor/BugsnagEditor.cs

```diff
@@ -192,6 +192,7 @@ private void DrawEnabledErrorTypesDropdown(BugsnagSettingsObject settings)
                 settings.EnabledErrorTypes.ANRs = EditorGUILayout.Toggle("ANRs (Android)", settings.EnabledErrorTypes.ANRs);
                 settings.EnabledErrorTypes.AppHangs = EditorGUILayout.Toggle("AppHangs (Cocoa)", settings.EnabledErrorTypes.AppHangs);
                 settings.EnabledErrorTypes.OOMs = EditorGUILayout.Toggle("OOMs (Cocoa)", settings.EnabledErrorTypes.OOMs);
+                settings.EnabledErrorTypes.Crashes = EditorGUILayout.Toggle("ThermalKills (Cocoa)", settings.EnabledErrorTypes.ThermalKills);
```

Comment:
That doesn't look right
```suggestion
                settings.EnabledErrorTypes.ThermalKills = EditorGUILayout.Toggle("ThermalKills (Cocoa)", settings.EnabledErrorTypes.ThermalKills);
```